pkgs/by-name/gx/gxml/package.nix

```diff
@@ -0,0 +1,58 @@
+{ fetchFromGitLab
+, glib
+, gobject-introspection
+, meson
+, ninja
+, pkg-config
+, stdenv
+, lib
+, libgee
+, libxml2
+, vala
+}:
+stdenv.mkDerivation rec {
```

Comment:
Would be nice to replace the `rec` with [`finalAttrs` pattern](https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes).